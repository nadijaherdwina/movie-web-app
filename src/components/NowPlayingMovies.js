import React from 'react';
import 'bootstrap/dist/css/bootstrap.min.css';
import '../App.css';


class NowPlayingMovies extends React.Component {
	constructor() {
		super();
		this.state = {
				id: '',
				poster: '',
				items: []
		}
	}
	componentDidMount() {
		let states = [];
		states.push({
			id: '1',
			title: 'Harry Potter and the Chamber of Secrets',
			poster: 'https://image.tmdb.org/t/p/w500/sdEOH0992YZ0QSxgXNIGLq1ToUi.jpg',
			releaseDate: '2002-11-13',
			description: 'Ignoring threats to his life, Harry returns to Hogwarts to investigate – aided by Ron and Hermione – a mysterious series of attacks.',
		});
		states.push({
			id: '2',
			title: 'Mamma Mia!',
			poster: 'https://image.tmdb.org/t/p/w500/gOm2iMMbC6EonrFzmSQ8xvCa4Ei.jpg',
			releaseDate: '2008-07-02',
			description: 'An independent, single mother who owns a small hotel on a Greek island is about to marry off the spirited young daughter she\'s raised alone. But, the daughter has secretly invited three of her mother\'s ex-lovers in the hopes of finding her biological father.',
		});
		states.push({
			id: '3',
			title: 'Harry Potter and the Chamber of Secrets',
			poster: 'https://image.tmdb.org/t/p/w500/sdEOH0992YZ0QSxgXNIGLq1ToUi.jpg',
			releaseDate: '2002-11-13',
			description: 'Ignoring threats to his life, Harry returns to Hogwarts to investigate – aided by Ron and Hermione – a mysterious series of attacks.',
		});
		states.push({
			id: '4',
			title: 'Mamma Mia!',
			poster: 'https://image.tmdb.org/t/p/w500/gOm2iMMbC6EonrFzmSQ8xvCa4Ei.jpg',
			releaseDate: '2008-07-02',
			description: 'An independent, single mother who owns a small hotel on a Greek island is about to marry off the spirited young daughter she\'s raised alone. But, the daughter has secretly invited three of her mother\'s ex-lovers in the hopes of finding her biological father.',
		});
		states.push({
			id: '3',
			title: 'Harry Potter and the Chamber of Secrets',
			poster: 'https://image.tmdb.org/t/p/w500/sdEOH0992YZ0QSxgXNIGLq1ToUi.jpg',
			releaseDate: '2002-11-13',
			description: 'Ignoring threats to his life, Harry returns to Hogwarts to investigate – aided by Ron and Hermione – a mysterious series of attacks.',
		});
		states.push({
			id: '4',
			title: 'Mamma Mia!',
			poster: 'https://image.tmdb.org/t/p/w500/gOm2iMMbC6EonrFzmSQ8xvCa4Ei.jpg',
			releaseDate: '2008-07-02',
			description: 'An independent, single mother who owns a small hotel on a Greek island is about to marry off the spirited young daughter she\'s raised alone. But, the daughter has secretly invited three of her mother\'s ex-lovers in the hopes of finding her biological father.',
		});
		states.push({
			id: '3',
			title: 'Harry Potter and the Chamber of Secrets',
			poster: 'https://image.tmdb.org/t/p/w500/sdEOH0992YZ0QSxgXNIGLq1ToUi.jpg',
			releaseDate: '2002-11-13',
			description: 'Ignoring threats to his life, Harry returns to Hogwarts to investigate – aided by Ron and Hermione – a mysterious series of attacks.',
		});
		states.push({
			id: '4',
			title: 'Mamma Mia!',
			poster: 'https://image.tmdb.org/t/p/w500/gOm2iMMbC6EonrFzmSQ8xvCa4Ei.jpg',
			releaseDate: '2008-07-02',
			description: 'An independent, single mother who owns a small hotel on a Greek island is about to marry off the spirited young daughter she\'s raised alone. But, the daughter has secretly invited three of her mother\'s ex-lovers in the hopes of finding her biological father.',
		});
		states.push({
			id: '3',
			title: 'Harry Potter and the Chamber of Secrets',
			poster: 'https://image.tmdb.org/t/p/w500/sdEOH0992YZ0QSxgXNIGLq1ToUi.jpg',
			releaseDate: '2002-11-13',
			description: 'Ignoring threats to his life, Harry returns to Hogwarts to investigate – aided by Ron and Hermione – a mysterious series of attacks.',
		});
		states.push({
			id: '4',
			title: 'Mamma Mia!',
			poster: 'https://image.tmdb.org/t/p/w500/gOm2iMMbC6EonrFzmSQ8xvCa4Ei.jpg',
			releaseDate: '2008-07-02',
			description: 'An independent, single mother who owns a small hotel on a Greek island is about to marry off the spirited young daughter she\'s raised alone. But, the daughter has secretly invited three of her mother\'s ex-lovers in the hopes of finding her biological father.',
		});
		states.push({
			id: '3',
			title: 'Harry Potter and the Chamber of Secrets',
			poster: 'https://image.tmdb.org/t/p/w500/sdEOH0992YZ0QSxgXNIGLq1ToUi.jpg',
			releaseDate: '2002-11-13',
			description: 'Ignoring threats to his life, Harry returns to Hogwarts to investigate – aided by Ron and Hermione – a mysterious series of attacks.',
		});
		states.push({
			id: '4',
			title: 'Mamma Mia!',
			poster: 'https://image.tmdb.org/t/p/w500/gOm2iMMbC6EonrFzmSQ8xvCa4Ei.jpg',
			releaseDate: '2008-07-02',
			description: 'An independent, single mother who owns a small hotel on a Greek island is about to marry off the spirited young daughter she\'s raised alone. But, the daughter has secretly invited three of her mother\'s ex-lovers in the hopes of finding her biological father.',
		});
		
		this.setState({
			items: states
		});
	}

	render() {
		return (
				<div className="align-items-center text-center justify-content-center vh-100 d-flex flex-column">
					<h1 className='display-3'>Movie List</h1>
					<div className='row movieDiv'>
						{this.state.items.map((item) => {
							return (
								<div className="col-md-2 col-sm-4 col-xs-6" > <img className="moviePosterList my-2" src={item.poster}/> </div>
							)
						})}
					</div>
				</div>

		);
	}
}

export default NowPlayingMovies;
