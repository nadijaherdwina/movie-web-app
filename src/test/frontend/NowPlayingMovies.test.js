import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, render } from 'enzyme';

import NowPlayingMovies from "../../components/NowPlayingMovies"
import { Link, MemoryRouter } from 'react-router';


Enzyme.configure({ adapter: new Adapter() });

describe('<NowPlayingMovies />', () => {
	it('renders without crashing', () => {
		const wrapper = shallow(<NowPlayingMovies />);
		expect(wrapper).toMatchSnapshot();
	});                  
        
});