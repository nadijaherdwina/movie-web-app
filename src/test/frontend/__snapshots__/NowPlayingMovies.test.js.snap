// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NowPlayingMovies /> renders without crashing 1`] = `ShallowWrapper {}`;

exports[`<NowPlayingMovies /> renders without crashing again 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {
          "class": "navbar navbar-expand-lg navbar-light bg-dark",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "navbar-brand",
              "href": "/",
            },
            "children": Array [
              Object {
                "data": " ",
                "next": Object {
                  "attribs": Object {
                    "class": "subtitle  pl-5 ml-5 pt-1",
                  },
                  "children": Array [
                    Object {
                      "data": " Home ",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "h5",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
              Object {
                "attribs": Object {
                  "class": "subtitle  pl-5 ml-5 pt-1",
                },
                "children": Array [
                  Object {
                    "data": " Home ",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "h5",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": Object {
                  "data": " ",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
            ],
            "name": "a",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "attribs": Object {
                "aria-controls": "navbarNav",
                "aria-expanded": "false",
                "aria-label": "Toggle navigation",
                "class": "navbar-toggler",
                "data-target": "#navbarNav",
                "data-toggle": "collapse",
                "type": "button",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "class": "navbar-toggler-icon",
                  },
                  "children": Array [],
                  "name": "span",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
              ],
              "name": "button",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "aria-controls": undefined,
                "aria-expanded": undefined,
                "aria-label": undefined,
                "class": undefined,
                "data-target": undefined,
                "data-toggle": undefined,
                "type": undefined,
              },
              "x-attribsPrefix": Object {
                "aria-controls": undefined,
                "aria-expanded": undefined,
                "aria-label": undefined,
                "class": undefined,
                "data-target": undefined,
                "data-toggle": undefined,
                "type": undefined,
              },
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
              "href": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
              "href": undefined,
            },
          },
          Object {
            "attribs": Object {
              "aria-controls": "navbarNav",
              "aria-expanded": "false",
              "aria-label": "Toggle navigation",
              "class": "navbar-toggler",
              "data-target": "#navbarNav",
              "data-toggle": "collapse",
              "type": "button",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "class": "navbar-toggler-icon",
                },
                "children": Array [],
                "name": "span",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
            ],
            "name": "button",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "navbar-brand",
                "href": "/",
              },
              "children": Array [
                Object {
                  "data": " ",
                  "next": Object {
                    "attribs": Object {
                      "class": "subtitle  pl-5 ml-5 pt-1",
                    },
                    "children": Array [
                      Object {
                        "data": " Home ",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "h5",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
                Object {
                  "attribs": Object {
                    "class": "subtitle  pl-5 ml-5 pt-1",
                  },
                  "children": Array [
                    Object {
                      "data": " Home ",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "h5",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": Object {
                    "data": " ",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
              ],
              "name": "a",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
                "href": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
                "href": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "aria-controls": undefined,
              "aria-expanded": undefined,
              "aria-label": undefined,
              "class": undefined,
              "data-target": undefined,
              "data-toggle": undefined,
              "type": undefined,
            },
            "x-attribsPrefix": Object {
              "aria-controls": undefined,
              "aria-expanded": undefined,
              "aria-label": undefined,
              "class": undefined,
              "data-target": undefined,
              "data-toggle": undefined,
              "type": undefined,
            },
          },
        ],
        "name": "nav",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {
            "class": "mx-3 mt-3 movieDiv header",
          },
          "children": Array [
            Object {
              "data": "Now Playing Movies",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "h2",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": Object {
            "attribs": Object {
              "class": "row movieDiv d-flex justify-content-center",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "class": "loader",
                },
                "children": Array [
                  Object {
                    "data": " ",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "div",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
            ],
            "name": "div",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": [Circular],
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
      Object {
        "attribs": Object {
          "class": "mx-3 mt-3 movieDiv header",
        },
        "children": Array [
          Object {
            "data": "Now Playing Movies",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "h2",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {
            "class": "row movieDiv d-flex justify-content-center",
          },
          "children": Array [
            Object {
              "attribs": Object {
                "class": "loader",
              },
              "children": Array [
                Object {
                  "data": " ",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "div",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
          ],
          "name": "div",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "class": "navbar navbar-expand-lg navbar-light bg-dark",
          },
          "children": Array [
            Object {
              "attribs": Object {
                "class": "navbar-brand",
                "href": "/",
              },
              "children": Array [
                Object {
                  "data": " ",
                  "next": Object {
                    "attribs": Object {
                      "class": "subtitle  pl-5 ml-5 pt-1",
                    },
                    "children": Array [
                      Object {
                        "data": " Home ",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "h5",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
                Object {
                  "attribs": Object {
                    "class": "subtitle  pl-5 ml-5 pt-1",
                  },
                  "children": Array [
                    Object {
                      "data": " Home ",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "h5",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": Object {
                    "data": " ",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
              ],
              "name": "a",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": Object {
                "attribs": Object {
                  "aria-controls": "navbarNav",
                  "aria-expanded": "false",
                  "aria-label": "Toggle navigation",
                  "class": "navbar-toggler",
                  "data-target": "#navbarNav",
                  "data-toggle": "collapse",
                  "type": "button",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "class": "navbar-toggler-icon",
                    },
                    "children": Array [],
                    "name": "span",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                ],
                "name": "button",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
                "x-attribsNamespace": Object {
                  "aria-controls": undefined,
                  "aria-expanded": undefined,
                  "aria-label": undefined,
                  "class": undefined,
                  "data-target": undefined,
                  "data-toggle": undefined,
                  "type": undefined,
                },
                "x-attribsPrefix": Object {
                  "aria-controls": undefined,
                  "aria-expanded": undefined,
                  "aria-label": undefined,
                  "class": undefined,
                  "data-target": undefined,
                  "data-toggle": undefined,
                  "type": undefined,
                },
              },
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
                "href": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
                "href": undefined,
              },
            },
            Object {
              "attribs": Object {
                "aria-controls": "navbarNav",
                "aria-expanded": "false",
                "aria-label": "Toggle navigation",
                "class": "navbar-toggler",
                "data-target": "#navbarNav",
                "data-toggle": "collapse",
                "type": "button",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "class": "navbar-toggler-icon",
                  },
                  "children": Array [],
                  "name": "span",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
              ],
              "name": "button",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "class": "navbar-brand",
                  "href": "/",
                },
                "children": Array [
                  Object {
                    "data": " ",
                    "next": Object {
                      "attribs": Object {
                        "class": "subtitle  pl-5 ml-5 pt-1",
                      },
                      "children": Array [
                        Object {
                          "data": " Home ",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "text",
                        },
                      ],
                      "name": "h5",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": [Circular],
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                      },
                    },
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                  Object {
                    "attribs": Object {
                      "class": "subtitle  pl-5 ml-5 pt-1",
                    },
                    "children": Array [
                      Object {
                        "data": " Home ",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "h5",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": Object {
                      "data": " ",
                      "next": [Circular],
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                ],
                "name": "a",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                  "href": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                  "href": undefined,
                },
              },
              "type": "tag",
              "x-attribsNamespace": Object {
                "aria-controls": undefined,
                "aria-expanded": undefined,
                "aria-label": undefined,
                "class": undefined,
                "data-target": undefined,
                "data-toggle": undefined,
                "type": undefined,
              },
              "x-attribsPrefix": Object {
                "aria-controls": undefined,
                "aria-expanded": undefined,
                "aria-label": undefined,
                "class": undefined,
                "data-target": undefined,
                "data-toggle": undefined,
                "type": undefined,
              },
            },
          ],
          "name": "nav",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
      Object {
        "attribs": Object {
          "class": "row movieDiv d-flex justify-content-center",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "loader",
            },
            "children": Array [
              Object {
                "data": " ",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "div",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "class": "mx-3 mt-3 movieDiv header",
          },
          "children": Array [
            Object {
              "data": "Now Playing Movies",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "h2",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": Object {
            "attribs": Object {
              "class": "navbar navbar-expand-lg navbar-light bg-dark",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "class": "navbar-brand",
                  "href": "/",
                },
                "children": Array [
                  Object {
                    "data": " ",
                    "next": Object {
                      "attribs": Object {
                        "class": "subtitle  pl-5 ml-5 pt-1",
                      },
                      "children": Array [
                        Object {
                          "data": " Home ",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "text",
                        },
                      ],
                      "name": "h5",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": [Circular],
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                      },
                    },
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                  Object {
                    "attribs": Object {
                      "class": "subtitle  pl-5 ml-5 pt-1",
                    },
                    "children": Array [
                      Object {
                        "data": " Home ",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "h5",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": Object {
                      "data": " ",
                      "next": [Circular],
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                ],
                "name": "a",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": Object {
                  "attribs": Object {
                    "aria-controls": "navbarNav",
                    "aria-expanded": "false",
                    "aria-label": "Toggle navigation",
                    "class": "navbar-toggler",
                    "data-target": "#navbarNav",
                    "data-toggle": "collapse",
                    "type": "button",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "class": "navbar-toggler-icon",
                      },
                      "children": Array [],
                      "name": "span",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                      },
                    },
                  ],
                  "name": "button",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "aria-controls": undefined,
                    "aria-expanded": undefined,
                    "aria-label": undefined,
                    "class": undefined,
                    "data-target": undefined,
                    "data-toggle": undefined,
                    "type": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "aria-controls": undefined,
                    "aria-expanded": undefined,
                    "aria-label": undefined,
                    "class": undefined,
                    "data-target": undefined,
                    "data-toggle": undefined,
                    "type": undefined,
                  },
                },
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                  "href": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                  "href": undefined,
                },
              },
              Object {
                "attribs": Object {
                  "aria-controls": "navbarNav",
                  "aria-expanded": "false",
                  "aria-label": "Toggle navigation",
                  "class": "navbar-toggler",
                  "data-target": "#navbarNav",
                  "data-toggle": "collapse",
                  "type": "button",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "class": "navbar-toggler-icon",
                    },
                    "children": Array [],
                    "name": "span",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                ],
                "name": "button",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "class": "navbar-brand",
                    "href": "/",
                  },
                  "children": Array [
                    Object {
                      "data": " ",
                      "next": Object {
                        "attribs": Object {
                          "class": "subtitle  pl-5 ml-5 pt-1",
                        },
                        "children": Array [
                          Object {
                            "data": " Home ",
                            "next": null,
                            "parent": [Circular],
                            "prev": null,
                            "type": "text",
                          },
                        ],
                        "name": "h5",
                        "namespace": "http://www.w3.org/1999/xhtml",
                        "next": null,
                        "parent": [Circular],
                        "prev": [Circular],
                        "type": "tag",
                        "x-attribsNamespace": Object {
                          "class": undefined,
                        },
                        "x-attribsPrefix": Object {
                          "class": undefined,
                        },
                      },
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                    Object {
                      "attribs": Object {
                        "class": "subtitle  pl-5 ml-5 pt-1",
                      },
                      "children": Array [
                        Object {
                          "data": " Home ",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "text",
                        },
                      ],
                      "name": "h5",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": Object {
                        "data": " ",
                        "next": [Circular],
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                      },
                    },
                  ],
                  "name": "a",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                    "href": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                    "href": undefined,
                  },
                },
                "type": "tag",
                "x-attribsNamespace": Object {
                  "aria-controls": undefined,
                  "aria-expanded": undefined,
                  "aria-label": undefined,
                  "class": undefined,
                  "data-target": undefined,
                  "data-toggle": undefined,
                  "type": undefined,
                },
                "x-attribsPrefix": Object {
                  "aria-controls": undefined,
                  "aria-expanded": undefined,
                  "aria-label": undefined,
                  "class": undefined,
                  "data-target": undefined,
                  "data-toggle": undefined,
                  "type": undefined,
                },
              },
            ],
            "name": "nav",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": [Circular],
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {},
    "x-attribsPrefix": Object {},
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
